Styles/ImageCowgirlOnWhiteHorse/index.js

```diff
@@ -0,0 +1,14 @@
+import { styled } from "styled-components";
+
+
+export function ImageCowgirlOnWhiteHorse(){
+return(
+
+    <ImageStyled src="/cowgirl_on_white_horse.jpg" alt="cowgirl on white horse" height={220} width={332}/>)
+}
```

Comment:
This is a super specific component, I think it would be less of an overhead for other programmers to understand your code if you just use your ImageStyled with the attributes directly where you would use this component. I would argue this does not provide enough simplification for it to become its own component.

Reply:
You're right. Thanks! :)